content/ADR-231-voting-power-usage-in-dapps

```diff
@@ -0,0 +1,39 @@
+---
+layout: adr
+adr: 231
+title: Voting Power usage for High Rated Places and Marketplace Saved Items
+date: 2023-05-09
+status: Idea
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - kevinszuchet
+---
+
+## Abstract
+
+This document outlines the guidelines implemented in Places and Marketplace to use the Voting Power (VP) against abuse. Given that both dApps have use cases where users can vote, rank, or save resources for later use, the Voting Power could help determine the number of reliable users interacting with them.
+
+## Context, Reach & Prioritization
+
+Since interacting with Places or the Marketplace only requires a wallet, creating multiple accounts can easily manipulate the number of likes a Place receives or the count of users who add an item to their list. To combat this, users' Voting Power can be utilized to address the problem of account spamming.
```

Comment:
What do you think about adding a `by` and an `anyone` to point that it's someone who can manipulate the number of likes by creating multiple accounts?
```suggestion
Since interacting with Places or the Marketplace only requires a wallet, by creating multiple accounts, anyone can easily manipulate the number of likes a Place receives or the count of users who add an item to their list. To combat this, users' Voting Power can be utilized to address the problem of account spamming.
```